service/common/src/main/java/org/apache/polaris/service/auth/DefaultOAuth2ApiService.java

```diff
@@ -75,43 +74,39 @@ public Response getToken(
     if (!tokenBroker.supportsRequestedTokenType(requestedTokenType)) {
```

Comment:
Why do we check `requestedTokenType`, but not use it?

Reply:
Yeah, we ought to update the `TokenBroker` API to take `requestedTokenType` as a method parameter.